setup.sh

```diff
@@ -298,13 +298,23 @@ function synchronize_files()
   setup_config_file
   ASSERT_IF_NOT_EQ_ZERO "Config file failed" "$?"
 
-  if [ -f "$HOME/.bashrc" ]; then
-      # Add to bashrc
-      echo "# $app_name" >> "$HOME/.bashrc"
-      echo "source $libdir/$BASH_AUTOCOMPLETE.sh" >> "$HOME/.bashrc"
-      update_path
+  if [ -f "$HOME/.bashrc" ] || [ -h "$HOME/.bashrc" ]; then
+    # Add to bashrc
+    echo "# $app_name" >> "$HOME/.bashrc"
+    echo "source $libdir/$BASH_AUTOCOMPLETE.sh" >> "$HOME/.bashrc"
+    update_path
   else
-      warning "Unable to find a shell."
+    warning "Unable to find a bash shell."
+  fi
+
+  if [ -f "$HOME/.zshrc" ] || [ -h "$HOME/.zshrc" ]; then 
+    # Add to zshrc
+    echo "# Enable bash completion for zsh" >> "$HOME/.zshrc"
+    echo "autoload bashcompinit && bashcompinit" >> "$HOME/.zshrc"
```

Comment:
Is this bashcompinit a default option in the zshrc?

Reply:
yes